src/main/java/org/jabref/gui/groups/GroupTreeView.java

```diff
@@ -399,4 +402,27 @@ private DroppingMouseLocation getDroppingMouseLocation(TreeTableRow<GroupNodeVie
             return DroppingMouseLocation.CENTER;
         }
     }
+
+    private class DragOverTreeItem {
```

Comment:
Maybe rename to `DragExpansionHandler` or something like that? (because it is not really a TreeItem but some kind of wrapper)